project-list/src/login.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+function LoginForm(props) {
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [loginError, setLoginError] = useState('');
+
+  function handleLoginSubmit(event) {
+    event.preventDefault();
+
+    // Static credentials 
+    const validUsername = 'admin@gmail.com';
+    const validPassword = 'password';
+
+    if (email === validUsername && password === validPassword) {
+      const accessToken = generateToken();
+      sessionStorage.setItem('accessToken', accessToken);
+      props.onLoginSuccess();
+    }
+    else {
+      setLoginError('Invalid username or password.');
+    }
+  }
+
+  function generateToken() {
+    const chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+    let token = '';
+    for (let i = 0; i < 20; i++) {
+      token += chars[Math.floor(Math.random() * chars.length)];
+    }
+    return token;
+  }
```

Comment:
You can create a cloud function on Netlify or on Vercel to handle authentication and use MongoDB to store username/password

Reply:
no need to this for now., we will do this in create-react-app